_operator_reference/deploy.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title: Deployment Environment Config Reference
+order: 5
+---
+
+{:.no_toc}
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+
+```yaml
+deploymentEnvironment:
+  size:
+  type:
+  accountName:
+  imageVariant:
+  bootstrapOnly:
+  updateVersions:
+  consul:
+    enabled:
+    address:
+  vault:
+    enabled:
+    address:
+  location:
+  customSizing:
+    spin-clouddriver:
+      replicas:
+      requests:
+    memory:
+    cpu:
+    limits:
+      memory:
+      cpu:
+  sidecars:
+    spin-clouddriver:
+    - name:
+      dockerImage:
+      port:
+      env:
+    abc:
+      args:
+      - arg1
+      command:
+      - arg1
+      configMapVolumeMounts:
+      - configMapName:
+    mountPath:
+      secretVolumeMounts:
+      - secretName:
+    mountPath:
+      mountPath:
+      securityContext:
+    privileged:
+  initContainers: {}
+  hostAliases: {}
+  nodeSelectors:
+    abc:
+  affinity: {}
+  tolerations:
+    spin-clouddriver:
+    - key:
+      operator:
+      value:
+      effect:
+  gitConfig:
+    upstreamUser:
+    originUser:
+  livenessProbeConfig:
+    enabled:
+    initialDelaySeconds:
+  haServices:
+    clouddriver:
+      enabled:
+      disableClouddriverRoDeck:
+      redisMasterEndpoint:
+      redisSlaveEndpoint:
+      redisSlaveDeckEndpoint:
+    echo:
+      enabled:
+```
+
+
+- `size`: SMALL
```

Comment:
should this be all caps?

Reply:
that's the way it is in the spinnakerservice.yml file